code/logger/logger.cpp

```diff
@@ -0,0 +1,34 @@
+#include "logger.h"
+
+#ifdef DEBUG
+
+#include <sstream>
+#include <ctime>
+
+LogFile Logger::file("logFile.txt");
+
+void Logger::Log(LogType type, const char* file, const char* func, int line, const std::string& msg) {
+	std::stringstream log;
+
+	// Adding timestamp
+	log << std::time(nullptr) << " ";
```

Comment:
```suggestion
	log << std::asctime(std::localtime(std::time(nullptr)) << " ";
```
So it's more human-friendly.

Reply:
Done

Reply:
The implemented utility for getting timestamp should be moved to some time utils in separate task.